python/tvm/relay/op/strategy/cuda.py

```diff
@@ -641,7 +642,7 @@ def conv3d_strategy_cuda(attrs, inputs, out_type, target):
                     (N % 16 == 0 and CI % 16 == 0 and CO % 16 == 0)
                     or (N % 8 == 0 and CI % 16 == 0 and CO % 32 == 0)
                     or (N % 32 == 0 and CI % 16 == 0 and CO % 8 == 0)
-                ):
+                ) and out_type == "float16":
```

Comment:
How about making this change to conv2d as well?. Perhaps better to send a separate PR for disabling fp32 tensorcore use since that would be a breaking change and potentially controversial.

Reply:
This was causing an issue with the new schedule, so I disabled it.